pkgs/top-level/all-packages.nix

```diff
@@ -18230,6 +18230,13 @@ with pkgs;
   apacheAnt = callPackage ../development/tools/build-managers/apache-ant { };
   ant = apacheAnt;
 
+  patchAntBuildfilesHook = makeSetupHook {
```

Comment:
About the name (patchAntBuildfilesHook), do you imagine that other kind of patching will happen here, or will this be a hook only for fixing the timestamp? (If the latter, maybe we could come up with a more specific name?)